src/lib.rs

```diff
@@ -150,10 +190,17 @@
 #![warn(missing_docs)]
 #![cfg_attr(feature = "nightly", feature(panic_handler))]
 
+#![cfg_attr(all(not(feature = "use_std"), not(test)), no_std)]
```

Comment:
In that case you can have an explicit `extern crate std` in the `mod test` and use the libstd prelude